components/lang/copy-libs-linux.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Add License Header

Reply:
Added. (Added to the other script as well)